### ISSUE TITLE
Unable to load context query for c_sharp

### ISSUE THREAD

**Issue by iryan2** [nvim-treesitter/nvim-treesitter-context#440]
### Description

When I open a `.cs` file in nvim I get 

```
Unable to load context query for c_sharp:
/opt/local/share/nvim/runtime/lua/vim/treesitter/query.lua:259: query: invalid node type at position 286 for language c_sharp
```

I've tried `TSInstall c_sharp` and `TSUpdate` which made no difference.

I apologise for not changing the below required fields. I'm using LazyVim, and I'm not familiar with how it uses `nvim-treesitter`. It'd be a whole exploratory task just to wire up the minimal example. 

I'm hoping in submitting this issue despite not meeting the requirements, one of the devs might be able to open a `.cs` file to see if they get the same error. If this is confirmed to not affect others, I'm happy to treat it as a "me problem" and close the issue.

### Neovim version

NVIM v0.9.5

### Actual behavior

As stated in description ^^

### Minimal config

```Lua
local plugins = {
  ts         = 'https://github.com/nvim-treesitter/nvim-treesitter',
  ts_context = 'https://github.com/nvim-treesitter/nvim-treesitter-context',
  -- ADD ADDITIONAL PLUGINS THAT ARE _NECESSARY_ TO REPRODUCE THE ISSUE
}

for name, url in pairs(plugins) do
  local install_path = '/tmp/nvim/site/'..name
  if vim.fn.isdirectory(install_path) == 0 then
    vim.fn.system { 'git', 'clone', '--depth=1', url, install_path }
  end
  vim.o.runtimepath = install_path..','..vim.o.runtimepath
end

-- ADD INIT.LUA SETTINGS THAT IS _NECESSARY_ FOR REPRODUCING THE ISSUE
```

### Steps to reproduce

1. `nvim --clean -u minimal.lua`
2. ...

**Comment by clason** [comment]
@amaanq

**Comment by StefanFanaru** [comment]
What I noticed, in c_sharp context.scm, for_each_statement should be foreach_statement (this is also broken in nvim-treesitter-textobjects)
And record_struct_declaration is also failing, but I don't know why.

As a workaround, until there is a proper fix, what I did, go to ~/.local/share/nvim/lazy/nvim-treesitter-context/queries/c_sharp/context.scm
change for_each_statement => foreach_statement
remove record_struct_decalation

**Comment by iryan2** [comment]
Thanks for the suggestion @StefanFanaru, I still get the same error after making those changes. Actually it was just one change `for_each_statement` -> `foreach_statement`; my `~/.local/share/nvim/lazy/nvim-treesitter-context/queries/c_sharp/context.scm` file didn't have any occurrences of `record_struct_decalation`.

**Comment by SerendipitousTesselation** [comment]
I had to revert #441 in my case. My case is a UWP code base. This is the error that got me here:

```
Unable to load context query for c_sharp:
...s/Neovim/share/nvim/runtime/lua/vim/treesitter/query.lua:279: Query error at 41:2. Invalid node type "foreach_statement":
(foreach_statement
 ^
```

**Comment by EliSauder** [comment]
I too also had to revert this change (and got the same error)

**Comment by EliSauder** [comment]
> I too also had to revert this change (and got the same error). I was in a worker service dotnet project not a UWP specific one.

Figured I would come back here and update this for people. After lots of ring around the rosy (started when I tried the min config and didn't have an issue). Turns out that nvim-treesitter hasn't been updating tags instead just releasing to master. I had specified that I should pull a version in updates. After telling it to pull directly from master, this was resolved.
